docs/BondingCurve.md

```diff
@@ -0,0 +1,170 @@
+# Bonding Curve Fundraising
+
+The bonding curve fundraising module enables projects and organizations to issue tokens and raise money to fund their vision. The core of this implementation is a **bonding curve**, or automated market maker - as conceptualized by individuals such as [Simon dela Rouviere](https://medium.com/@simondlr/tokens-2-0-curved-token-bonding-in-curation-markets-1764a2e0bee5), [Billy Rennekamp](https://medium.com/@billyrennekamp/converting-between-bancor-and-bonding-curve-price-formulas-9c11309062f5) and [Thibauld Favre](https://github.com/C-ORG/whitepaper) to enable continuous funding for organizations, coupled with guaranteed liquidity for investors without relying on exchanges.
+
+Tokens issued via bonding curves can offer rights in the organization, such as **dividends** on future revenue or **governance rights** through token locking.
+
+This type of fundraising might allow for more flexibility, accountability, and alignment of incentives than alternative methods (such as ICOs or private fundraising).
+
+## How it works
+
+- Anyone can deposit a specified collateral token (such as DAI) to purchase bonded tokens.
+- The **bonding curve** is an automated market maker contract that mints tokens to buyers at an algorithmically determined price. The automated market maker allows users to buy or sell tokens at any time for known prices with minimal slippage.
+- When a user **buys** bonded tokens: Collateral tokens are split between the beneficiary treasury (to fund the organization) and the reserve (to facilitate liquidity for sells). The buyer receives bonded tokens based on the current price.
```

Comment:
To be more clear, think it would be good to say something like "When a user buys bonded tokens with collateral tokens. . . "